simple_salesforce/sfdc_session.py

```diff
@@ -0,0 +1,81 @@
+import re
+
+from requests import Session
+from xml.etree import ElementTree as ET
+
+
+class SfdcSession(Session):
```

Comment:
Any reason you re-wrote the login class instead of using the existing login one? I'd rather keep it consolidated into a single class unless absolutely necessary.

Reply:
I had another login class because the code from sfdc_lib depended on a slightly different implementation of a "session". If you figured out a way to get the two things to work congruously that would be great, agreed that it should ideally be consolidated!